processing/src/main/java/io/druid/query/search/SearchQueryRunner.java

```diff
@@ -71,6 +73,12 @@
 public class SearchQueryRunner implements QueryRunner<Result<SearchResultValue>>
 {
   private static final EmittingLogger log = new EmittingLogger(SearchQueryRunner.class);
+
+  private static final double HIGH_FILTER_SELECTIVITY_THRESHOLD_FOR_CONCISE = 0.99;
```

Comment:
Actually reading more over this code, I think this might belong in the bitmap factories

@gianm thoughts?